tests/test_testing.py

```
@@ -87,3 +125,126 @@ def test_time_generator():
+    capfire.reconfigure(metrics=False)
+
+    logfire.metric_counter('a_counter').add(1)
+    capfire.metrics_reader.collect()
+
+    assert capfire.metrics_reader.get_metrics_data() is None
+
+
+def test_reconfigure_replaces_scrubbing(capfire: CaptureLogfire) -> None:
+    capfire.reconfigure(scrubbing=False)
+
+    logfire.info('hi', password='hunter2')
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)